distributions/demo-google-deployment/api/build.gradle

```diff
@@ -63,20 +63,40 @@ addCloudExtensionDependency(project)
 
 mainClassName = 'org.dataportabilityproject.api.ApiMain'
 
-task copyEnvResources(type: Copy) {
-    description = 'Copies environment-specific resources and configuration into the jar, ' +
-            'including index.html'
+task copyIndexHtml(type: Copy) {
+    description = 'Copies index.html into the jar'
+    ext.env = project.hasProperty('env') ? env : 'local'
+    into 'src/main/resources/config'
```

Comment:
Does the build_and_deploy_static_content script move the index file in resource/config/enviornments?

Reply:
Good catch, it does not, just updated it

Reply:
Later on down the line, we will probably not include this for RPC only serving so we may want to make it optional

Reply:
Ack. I'd rather figure that out in a separate distribution, leave this one as an HTTP-serving one